src/containers/TasksByDayPage/components/TabContainer/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+import Typography from '@material-ui/core/Typography'
+import styled from 'styled-components'
+
+export default function TabContainer(props) {
+  return (<StyledTypography component="div">{props.children}</StyledTypography>);
```

Comment:
```suggestion
  return <StyledTypography component="div">{children}</StyledTypography>
```
nếu viết được inline thì ko cần ngoặc tròn nữa